### PR TITLE
Update footer.html - Adding Fitur Share Website Fossasia

### PR DESCRIPTION
**Pull request by amryarios**
Adding Fitur Share Website Fossasia in Social Media
- Facebook
- Twitter
- Google+

Screenshot ;
- Share Google + 

- Share Twitter

- Share Facebook

Preview Link : https://amryarios.github.io/gci17.fossasia.org/

Thank You :+1:

### CODE REVIEW

**Comment by amryarios** [comment]
Ping @fossasia/website-admins

**Review by anshumanv — COMMENTED** [review]
Why is this needed and which issue does it solve?

File: _includes/footer.html [hunk 1]
```diff
@@ -8,59 +8,20 @@
...
+									<a href="https://www.facebook.com/sharer.php?u=https://gci17.fossasia.org/" target="_blank">
+										<i class="fa fa-facebook"></i>acebook
 									</a>
 								</li>
 								<li>
-									<a href="https://www.vk.com/fossasia">
-										<i class="fa fa-vk" aria-hidden="true"></i>
-									</a>
-								</li>
-								<li>
```
Comment (0x48piraj) [inline _includes/footer.html:56]: Why deleted and further not added ?

File: _includes/footer.html [hunk 2]
```diff
@@ -8,59 +8,20 @@
...
-									<a href="https://plus.google.com/108920596016838318216">
-										<i class="fa fa-google" aria-hidden="true"></i>
-									</a>
-								</li>
-								<li>
-									<a href="https://www.linkedin.com/groups/FOSSASIA-3762811">
-										<i class="fa fa-linkedin" aria-hidden="true"></i>
```
Comment (0x48piraj) [inline _includes/footer.html:28]: Please tell the reason for doing this

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Look at my code reviews.
Very messy PR

**Pull request closed without merging**